### ISSUE TITLE
PDF Component Clear Event Not Updating value to None

### ISSUE THREAD

**Issue by Jocelyne-Smith** [freddyaboulton/gradio-pdf#22]
The clear function of the Gradio PDF component does not update the value to None when called. Therefore appropriate error handling cannot take place.

**Comment by freddyaboulton** [comment]
Can you post a code example to show what you mean @Jocelyne-Smith ? Sorry for the delay.

**Comment by Jocelyne-Smith** [comment]
```
import gradio as gr
from gradio_pdf import PDF

def check_upload(file):
        print(f"Changed: {file}")

# Gradio Interface
with gr.Blocks() as interface:
    file_input = PDF()
    btn_input = gr.Button()
    
    btn_input.click(fn=check_upload, inputs=file_input)

interface.launch()

```

I uploaded a file and cleared it, but the value remained after it was cleared. Therefore, appropriate error handling can't take place.

https://github.com/user-attachments/assets/af215af3-3371-423d-873a-fc62a65e4201

**Comment by freddyaboulton** [comment]
Fixed in version 0.0.18 - thanks @Jocelyne-Smith !
